extensions/cxf/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: "CXF"
+metadata:
+  keywords:
+  - "cxf"
+  - "jaxws"
+  - "jax-ws"
+  - "web"
+  - "soap"
+  categories:
+  - "web"
+  - "reactive"
```

Comment:
Did it support reactive types ?
If yes some example in the documentation would be great.

Reply:
Cxf provide a async annotation. But not sure it’s well integrated and it’s works. It must be test. I will test it. To check if it’s ok

Reply:
I think we should remove the `reactive` one. It's not really worth adding it there IMHO.